corehq/apps/hqwebapp/management/commands/build_requirejs.py

```diff
@@ -96,29 +96,27 @@ def _relative(path, root=None):
                 ...
             }
             '''
-            dirs = {}
+            dirs = defaultdict(set)
             for filename in html_files:
-                proc = subprocess.Popen(["grep", "^\s*{% requirejs_main [^%]* %}\s*$", filename], stdout=subprocess.PIPE)
+                proc = subprocess.Popen(["grep", "^\s*{% requirejs_main [^%]* %}\s*$", filename],
+                                        stdout=subprocess.PIPE)
                 (out, err) = proc.communicate()
                 if out:
                     match = re.search(r"{% requirejs_main .(([^%]*)/[^/%]*). %}", out)
                     if match:
                         main = match.group(1)
                         directory = match.group(2)
                         if os.path.exists(os.path.join(self.root_dir, 'staticfiles', main + '.js')):
-                            if directory not in dirs:
-                                dirs.update({directory: set()})
                             dirs[directory].add(main)
 
-
             # For each directory, add an optimized "module" entry including all of the main modules in that dir.
             # For each of these entries, r.js will create an optimized bundle of these main modules and all their
             # dependencies
             for directory, mains in dirs.items():
                 config['modules'].append({
                     'name': os.path.join(directory, "bundle"),
                     'exclude': ['hqwebapp/js/common', 'hqwebapp/js/base_main'],
-                    'include': list(mains),
+                    'include': sorted(list(mains)),
```

Comment:
nit: `list()` not necessary and just makes the interpreter do extra work. `sorted(mains)` will work just fine